test/test_puma_server.rb

```diff
@@ -326,16 +326,16 @@ def test_early_hints_works
 
     data = send_http_and_read "HEAD / HTTP/1.0\r\n\r\n"
 
-    expected_data = (<<EOF
-HTTP/1.1 103 Early Hints
-Link: </style.css>; rel=preload; as=style
-Link: </script.js>; rel=preload
+#    expected_data = <<~EOF.split("\n").join("\r\n") + "\r\n\r\n"
```

Comment:
No need for the commented code?

Reply:
Sorry, the multi-tasking problem...

Fixed